ioctl/newcmd/account/account.go

```diff
@@ -0,0 +1,403 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	ecrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/status"
+
+	"github.com/iotexproject/go-pkgs/crypto"
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/cmd/hdwallet"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/ioctl/validator"
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+)
+
+// Multi-language support
+var (
+	accountCmdShorts = map[config.Language]string{
+		config.English: "Manage accounts of IoTeX blockchain",
+		config.Chinese: "管理IoTeX区块链上的账号",
+	}
+	accountCmdUses = map[config.Language]string{
+		config.English: "account",
+		config.Chinese: "账户",
+	}
+	flagEndpoint = map[config.Language]string{
+		config.English: "set endpoint for once",
+		config.Chinese: "一次设置端点",
+	}
+	flagInsecure = map[config.Language]string{
+		config.English: "insecure connection for once",
+		config.Chinese: "一次不安全连接",
+	}
+)
+
+// Errors
+var (
+	ErrPasswdNotMatch = errors.New("password doesn't match")
+)
+
+// CryptoSm2 is a flag for sm2 cryptographic algorithm
+var CryptoSm2 bool
```

Comment:
put it into ioctl.Client

Reply:
leave it in this pr and i will do it in a new pr  if  required.